src/destination/itm.rs

```diff
@@ -1,33 +1,42 @@
 //! ITM module
 
-use cortex_m::{itm};
+use cortex_m::{itm, interrupt::Mutex};
 use core::fmt;
 
+use core::cell::RefCell;
+
 /// ITM based destination
 pub struct Itm {
-    inner: cortex_m::peripheral::ITM,
+    inner: Mutex<RefCell<cortex_m::peripheral::ITM>>,
```

Comment:
`RefCell` is unnecessary overhead

Reply:
Cortex-m mutex does not provide a mutable reference and instead relies on interior mutability - https://docs.rs/cortex-m/0.5.8/cortex_m/interrupt/struct.Mutex.html#method.borrow. Hence the RefCell is required.